### ISSUE TITLE
Add config option to use system CA bundle

### ISSUE THREAD

**Issue by wrycu** [kennethreitz/requests#2500]
I'm using requests with an internal CA which is trusted on my machines.  I recently discovered that you can modify the code in certs.py to return the system CA bundle (thus allowing verified HTTPS requests), but I think this should really be an option, not a modification to the module.

And since we all use venvs to dev, simply adding it to the Requests bundle is not a very elegant solution.

**Comment by sigmavirus24** [comment]
If you know the location, you can use the verify parameter on a session to set the location or you can use `REQUESTS_CA_BUNDLE` and `CURL_CA_BUNDLE`.

We used to have a way to auto-discover system bundles but there were far too many corner cases.

**Comment by Lukasa** [comment]
This is another thing that can be solved by shazow/urllib3#507, if it gets merged. Until that time, I agree that this is more complicated than simply passing your own certificate bundle to requests.

For anyone who is interested, you can build this bundle by running:

``` bash
$ cat `python -c 'import requests; print requests.certs.where()'` <your PEM file>
```
